--- kulic-1.1/src/main.cc~	2009-12-20 20:08:47.000000000 +0100
+++ kulic-1.1/src/main.cc	2009-12-29 14:13:20.153169940 +0100
@@ -23,7 +23,7 @@
 #include <strings.h>
 #include <errno.h>
 #include <sys/file.h>
-
+#include <cstring>
 
 
 
--- kulic-1.1/src/GMENUSET.cpp~	2009-12-19 14:46:07.000000000 +0100
+++ kulic-1.1/src/GMENUSET.cpp	2009-12-29 14:14:08.279831763 +0100
@@ -5,6 +5,7 @@
 #include "stdafx.h"
 #include "GMenuSet.h"
 #include <iostream>
+#include <cstring>
 
 using namespace std;
 
--- kulic-1.1/src/GRUN.cpp~	2009-12-20 19:46:28.000000000 +0100
+++ kulic-1.1/src/GRUN.cpp	2009-12-29 14:15:08.976492461 +0100
@@ -9,6 +9,7 @@
 #include "GMap.h"
 #include "GRun.h"
 #include "zbrane.h"
+#include <cstring>
 
 //#include "MultiPlay.h"
 
--- kulic-1.1/src/GVIEW.cpp~	2009-12-19 18:50:45.000000000 +0100
+++ kulic-1.1/src/GVIEW.cpp	2009-12-29 14:15:52.253154998 +0100
@@ -9,6 +9,7 @@
 #include "GView.h"
 #include "GMap.h"
 #include "GRun.h"
+#include <cstring>
 
 
 #define DXTABLE    900
