--- kulic-1.1-orig/src/GMAP.cpp	2009-12-29 15:53:05.699238367 +0100
+++ kulic-1.1/src/GMAP.cpp	2009-12-29 15:55:54.510065847 +0100
@@ -443,9 +443,9 @@
 
 	sprintf(s, "maps/map%d.wav", mapa);
 	if ((m_music = load_wav(s)) == NULL) {
-		sprintf(s, "/usr/local/share/kulic/maps/map%d.wav", mapa);
+		sprintf(s, "%s/maps/map%d.wav", PACKAGE_DATA_DIR, mapa);
 		if ((m_music = load_wav(s)) == NULL) {
-			sprintf(s, "/usr/share/kulic/maps/map%d.wav", mapa);
+			sprintf(s, "%s/maps/map%d.wav", PACKAGE_DATA_DIR, mapa);
 			m_music = load_wav(s);
 		}
 	}
--- kulic-1.1-orig/src/HGFX.cpp	2009-12-29 15:53:05.696945688 +0100
+++ kulic-1.1/src/HGFX.cpp	2009-12-29 15:59:16.616075254 +0100
@@ -42,11 +42,11 @@
 	bmp = load_bitmap(filename, pal);
 	if (bmp == NULL) {
 		char filename2[255];
-		sprintf(filename2, "/usr/local/share/kulic/%s", filename);
+		sprintf(filename2, "%s/%s", PACKAGE_DATA_DIR, filename);
 		bmp = load_bitmap(filename2, pal);
 		
 		if (bmp == NULL) {
-			sprintf(filename2, "/usr/share/kulic/%s", filename);
+			sprintf(filename2, "%s/%s", PACKAGE_DATA_DIR, filename);
 			bmp = load_bitmap(filename2, pal);
 		}
 	}
	
--- kulic-1.1-orig/src/GMenuBack.cpp	2009-12-29 15:53:05.699238367 +0100
+++ kulic-1.1/src/GMenuBack.cpp	2009-12-29 17:27:23.268671770 +0100
@@ -43,8 +43,7 @@
 
 	if (ini.hudba)
 		if ((m_dat = load_datafile("sfx/menu.dat")) == NULL)
-			if ((m_dat = load_datafile("/usr/local/share/kulic/sfx/menu.dat")) == NULL)
-				m_dat = load_datafile("/usr/share/kulic/sfx/menu.dat");
+			m_dat = load_datafile(PACKAGE_DATA_DIR "/sfx/menu.dat");
 
 	
 	
--- kulic-1.1-orig/src/GRUN.cpp	2009-12-29 15:53:05.699238367 +0100
+++ kulic-1.1/src/GRUN.cpp	2009-12-29 17:30:45.285526484 +0100
@@ -279,11 +279,10 @@
 
 	fonts = load_datafile("gfx/view/fonts.dat");
 	if (fonts == NULL) 
-		if ((fonts = load_datafile("/usr/local/share/kulic/gfx/view/fonts.dat")) == NULL)
-			if ((fonts = load_datafile("/usr/share/kulic/gfx/view/fonts.dat")) == NULL) {
-				cerr << "chybi fonts.dat" << endl;
-				return false;
-			}
+	    if ((fonts = load_datafile(PACKAGE_DATA_DIR "/gfx/view/fonts.dat")) == NULL) {
+		    cerr << "chybi fonts.dat" << endl;
+		    return false;
+	    }
 
 	if (!m_sshoots.LoadSFX()) {
 		cerr << "Chybi soubor shoots.dat" << endl;
--- kulic-1.1-orig/src/GSShoots.cpp	2009-12-29 15:53:05.699238367 +0100
+++ kulic-1.1/src/GSShoots.cpp	2009-12-29 17:36:58.131947319 +0100
@@ -24,9 +24,8 @@
 {
 	if (!ini.zvuky) return true;
 	if((m_soundsatd = load_datafile("sfx/shoots.dat")) == NULL) 
-		if((m_soundsatd = load_datafile("/usr/local/share/kulic/sfx/shoots.dat")) == NULL) 
-			if((m_soundsatd = load_datafile("/usr/share/kulic/sfx/shoots.dat")) == NULL) 
-				return true;
+		    if((m_soundsatd = load_datafile(PACKAGE_DATA_DIR "/sfx/shoots.dat")) == NULL) 
+			    return true;
 
 	for (int i = 0; i < MAX_SAMPLES; i++)
 		if((m_voices[i] = allocate_voice((SAMPLE *)m_soundsatd[i].dat)) == -1) 
